common/partition/interface.go

```diff
@@ -30,15 +30,17 @@ import (
 	"github.com/uber/cadence/common/dynamicconfig"
 )
 
-// Config is the base configuration for the partitioning library
-type Config struct {
-	IsolationGroupEnabled dynamicconfig.BoolPropertyFnWithDomainFilter
-	AllIsolationGroups    []string
-}
-
 type Partitioner interface {
 	// GetIsolationGroupByDomainID gets where the task workflow should be executing. Largely used by Matching
 	// when determining which isolationGroup to place the tasks in.
 	// Implementations ought to return (nil, nil) for when the feature is not enabled.
 	GetIsolationGroupByDomainID(ctx context.Context, DomainID string, partitionKey PartitionConfig) (*string, error)
```

Comment:
In this method, we're using `IsolationGroupEnabled` with domainName as filter, but what we pass is `domainID`

Reply:
yes, the domainID is being resolved to name in `AvailableIsolationGroupsByDomainID`